ui/main/src/app/modules/feedconfiguration/feedconfiguration.component.html

```diff
@@ -14,7 +14,16 @@
 
     <form [formGroup]="feedConfigurationForm">
         <div *ngFor="let processGroup of processGroupsAndLabels; " class="opfab-feedconfiguration-processlist">
-            <h5 class="font-weight-bold">{{processGroup.groupLabel}}</h5>
+            <label class="opfab-checkbox" *ngIf="! isAllProcessesSelectedPerProcessGroup.get(processGroup.groupId)">
+                <h5 class="font-weight-bold">{{processGroup.groupLabel}}</h5>
+                <input type="checkbox" (click)="selectAllProcesses(processGroup.groupId)">
+                <span class="opfab-checkbox-checkmark"></span>
+            </label>
+            <label class="opfab-checkbox" *ngIf="isAllProcessesSelectedPerProcessGroup.get(processGroup.groupId)">
+                <h5 class="font-weight-bold">{{processGroup.groupLabel}}</h5>
+                <input type="checkbox" (click)="unselectAllProcesses(processGroup.groupId)" checked>
+                <span class="opfab-checkbox-checkmark"></span>
+            </label>
```

Comment:
could it be better to avoid the double ngIf by calling a single method on click "toggleSelctAllProcesses(processGroup.groupId)" and bind the [checked]="isAllProcessesSelectedPerProcessGroup.get(processGroup.groupId)" ?

Reply:
Done